reviews/templates/mail/concept_other_applicants.html

```diff
@@ -0,0 +1,23 @@
+{% extends "mail/base.html" %}
+
+{% block content_nl %}
+Beste collega,<br />
+<br />
+Uw collega {{ creator }} heeft een concept-aanmelding ingediend bij de ethische commissie voor een onderzoek, waar u aan meewerkt.<br />
+Uw concept-aanmelding wordt door de eindverantwoordelijk onderzoeker {{ supervisor }} van deze aanvraag gecontroleerd.
```

Comment:
I would say 'De aanmelding wordt eerst door de eindverantwoordelijk onderzoeker {{ supervisor }} van deze aanvraag gecontroleerd.' 

Mostly because I think 'your' is a tad odd here. Technically it's true, they are named as an applicant, but this email exists because some people were complaining they weren't informed someone (wrongly) named them as an applicant.